playbooks/openstack-cloud-controller-manager-acceptance-test-lb-octavia-nested/playbook-in-vm/run.yaml

```diff
@@ -0,0 +1,190 @@
+              sudo systemctl restart docker
+              sudo systemctl status docker
+              sudo ifconfig -a
+          fi
+          docker --version
+          # Get the latest stable version of kubernetes
+          echo "Starting docker service"
+          sudo systemctl enable docker.service
+          sudo systemctl start docker.service --ignore-dependencies
+          echo "Checking docker service"
+          sudo docker ps
+          wget -c https://github.com/coreos/etcd/releases/download/v3.3.0/etcd-v3.3.0-linux-amd64.tar.gz
+          tar xzvf etcd-v3.3.0-linux-amd64.tar.gz
+          cp etcd-v3.3.0-linux-amd64/etcd /usr/local/bin/
+          cp etcd-v3.3.0-linux-amd64/etcdctl /usr/local/bin/
+        executable: /bin/bash
+      environment:
+        GOPATH: '{{ ansible_user_dir }}'
+        PATH: '/usr/local/go/bin:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}'
+
+- name: Set up Kubernetes local cluster
+  hosts: "{{ server_name }}"
+  become: yes
+  gather_facts: no
+  tasks:
+   - name: Set up Kubernetes local cluster
+     block:
+       - shell:
+           cmd: |
+             set -e
+             set -x
+             export PATH="/usr/local/go/bin:{{ ansible_user_dir }}/bin:${PATH}"
+             export GOPATH="{{ ansible_user_dir }}"
+             source openrc
+             mkdir -p /home/ubuntu/.kube
+
+             # Debuging ...
+             export API_HOST_IP="172.17.0.1"
+             export KUBELET_HOST="0.0.0.0"
+             export WAIT_FOR_URL_API_SERVER=120
+
+
+             echo "Stopping firewall and allow all traffic..."
+             iptables -F
+             iptables -X
+             iptables -t nat -F
+             iptables -t nat -X
+             iptables -t mangle -F
+             iptables -t mangle -X
+             iptables -P INPUT ACCEPT
+             iptables -P FORWARD ACCEPT
+             iptables -P OUTPUT ACCEPT
+
+             export ALLOW_SECURITY_CONTEXT=true
+             export ENABLE_CRI=false
+             export ENABLE_HOSTPATH_PROVISIONER=true
+             export ENABLE_SINGLE_CA_SIGNER=true
+             # export KUBE_ENABLE_CLUSTER_DASHBOARD=true
+             export KUBE_ENABLE_CLUSTER_DNS=false
+             export LOG_LEVEL=4
+
+             # go where we cloned kubernetes repository
+             cd $GOPATH/src/k8s.io/kubernetes/
+
+             # we want to use the openstack cloud provider
+             export CLOUD_PROVIDER=openstack
+             # we want to run a separate cloud-controller-manager for openstack
+             export EXTERNAL_CLOUD_PROVIDER=true
+             # DO NOT change the location of the cloud-config file. It's important for the old cinder provider to work
+             export CLOUD_CONFIG=/etc/kubernetes/cloud-config
+             # specify the OCCM binary
+             export EXTERNAL_CLOUD_PROVIDER_BINARY="{{ ansible_user_dir }}/src/github.com/theopenlab/openstack-cloud-controller-manager/openstack-cloud-controller-manager"
```

Comment:
Should not hard code project path "theopenlab/openstack-cloud-controller-manager", it might be "dims/openstack-cloud-controller-manager" or "kubernetes/cloud-provider-openstack" in the future, could we search the binary "openstack-cloud-controller-manager" in $GOPATH/src, then get absolute path?